frontend/src/pages/Contributions.jsx

```
@@ -167,44 +363,81 @@ function Contributions() {
+                        <p className="price">${contribution.price}</p>
+                        <div className="rating">
+                          {contribution.rating ? `★ ${contribution.rating}` : 'Not rated yet'}
+                        </div>
+                        {isUserAuthenticated && isEnrolled(contribution.id) ? (
+                          <div className="enrolled-indicator">
+                            <span className="enrolled-badge">Enrolled</span>
+                            <button 
+                              onClick={() => handleViewClick(contribution.id)}
+                              className="view-enrolled-btn"
+                            >
+                              View Content
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)